x-pack/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/external/archive_maintenance_window.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import expect from '@kbn/expect';
+import { UserAtSpaceScenarios } from '../../../../scenarios';
+import { getUrlPrefix, ObjectRemover } from '../../../../../common/lib';
+import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function updateMaintenanceWindowTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+  describe('archiveMaintenanceWindow', () => {
+    const objectRemover = new ObjectRemover(supertest);
```

Comment:
Shall we include validations test here?

Reply:
Do you mean like "bad request"? I should test if the body does not have the right format. 👍 

Is there anything else missing?

Reply:
yeah bad request validations, invalid params and body. I guess that's it.